src/ng-xform/fields-group/fields-group.component.html

```diff
@@ -0,0 +1,36 @@
+<div *ngFor="let field of fields">
+    <div [ngSwitch]="field.controlType">
+
+        <ng-template [ngSwitchCase]="'GROUP'">
+            <hr *ngIf='field.label'>

```

Comment:
Don't you need to close this tag?

Reply:
No, it is a horizontal line, it does not have content